src/Microsoft.IdentityModel.Tokens/LogMessages.cs

```diff
@@ -36,7 +36,7 @@ internal static class LogMessages
         public const string IDX10208 = "IDX10208: Unable to validate audience. validationParameters.ValidAudience is null or whitespace and validationParameters.ValidAudiences is null.";
         public const string IDX10209 = "IDX10209: Token has length: '{0}' which is larger than the MaximumTokenSizeInBytes: '{1}'.";
         public const string IDX10211 = "IDX10211: Unable to validate issuer. The 'issuer' parameter is null or whitespace.";
-        public const string IDX10214 = "IDX10214: Audience validation failed. Audiences: '{0}'. Did not match: validationParameters.ValidAudience: '{1}' or validationParameters.ValidAudiences: '{2}'.";
+        public const string IDX10214 = "IDX10214: Audience validation failed. Audiences: '{0}'. Did not match: validationParameters.ValidAudiences: '{1}'.";
```

Comment:
This log message is used by the current implementation. Removing the 3rd parameter would drop information from the logs for all of our current users.

Please avoid modifying existing log messages and create new ones if they are needed.
The fact that this did not fail a test should make us extra wary when making changes to anything that is not exclusive to our new code path.

Reply:
Great catch, updated. Thanks!